src/client/containers/view-batch-detail-page/view-batch-detail-page.component.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ViewBatchDetails from '../../components/view-batch-detail/view-batch-details.component';
+import SidebarMenu from '../../components/side-navigation/sidebar.component';
+import EditBatchImage from '../../components/edit-batch-image/edit-batch-image.component';
+import DeleteBatch from '../../components/delete-confim/batch-delete-button.component';
+import Footer from '../../components/footer/footer.component';
+import image from '../../assets/images/crop-image.png';
+import penImage from '../../assets/images/edit-icon.png';
+import './view-batch-detail-page.style.css';
+// import { getTokenWithHeaders } from '../../firebase/getTokenWithHeaders';
+
+const BatchDetailPage = () => {
+  // const [crop, setCrop] = useState([]);
+  // const [batch, setBatch] = useState([]);
+
+  // const batchId = 1;
+
+  // // Use this function to fetch APIs
+  // const getData = async () => {
+  //   // fetch batch api
+  //   const headers = await getTokenWithHeaders();
+  //   const rbatch = await fetch(`/api/batches/${batchId}`, {
+  //     method: 'GET',
+  //     headers,
+  //   }).then((data) => data.json());
+  //   console.log(rbatch);
+  //   setBatch(rbatch);
+
+  //   // fetch crop api
+  //   const rcrop = await fetch(
+  //     `/api/crops/${batch[0].fk_crop_id}`,
+  //     {
+  //       method: 'GET',
+  //       headers,
+  //     },
+  //   ).then((data) => data.json());
+  //   console.log(rcrop);
+  //   setCrop(rcrop);
+  // };
+
+  // getData();
```

Comment:
Above is currently not working, so hard-coded values are used below. Question is if we can merge this or fix this given how close we are to deadline?

Reply:
@zkwsk  it can work, I tried it out 🤠